internal/zero/controller/usage_reporter.go

```diff
@@ -0,0 +1,278 @@
+package controller
```

Comment:
(nit) `controller` package currently only has top-level logic, while other packages reside in their dedicated packages - can we move the usage reporter into some other package for consistency?

Reply:
I don't understand the objection, but ok.